content/document.js

```diff
@@ -230,25 +236,29 @@ function update(folder, rawHTML, metadata) {
         newChildSlug
       );
       saveHTMLFile(fileInfo.path, rawHTML, metadata);
+      redirects.set(
+        buildURL(childLocale, oldChildSlug),
+        buildURL(childLocale, newChildSlug)
+      );
     }
+    redirects.set(buildURL(locale, oldSlug), buildURL(locale, newSlug));
     const newFolderPath = buildPath(
-      path.join(CONTENT_ROOT, metadata.locale.toLowerCase()),
+      path.join(CONTENT_ROOT, locale.toLowerCase()),
       newSlug
     );
+    const oldFolderPath = buildPath(
+      path.join(CONTENT_ROOT, locale.toLowerCase()),
+      oldSlug
+    );
 
     // XXX we *could* call out to a shell here and attempt
     // to execute `git mv $folder $newFolder` and only if that didn't work
     // do we fall back on good old `fs.renameSync`.
-    fs.renameSync(folderPath, newFolderPath);
+    fs.renameSync(oldFolderPath, newFolderPath);
```

Comment:
On Slack you said:
"Also we should make sure that git is actually able to track if files just move rather than the are deleted and recreated (looking into that)."

I think that's basically what the (`XXX ...`) comment above is about. 
Having to do:

```
git status 
git add files/en-us/foo/new/one
git add files/en-us/foo/new/two
git add files/en-us/foo/new/three
git commit -a -m "renamed foo/old to foo/new"
```
is more annoying that just
```
git status 
git commit -a -m "renamed foo/old to foo/new"
```
...if Yari had used `git mv files/en-us/old files/en-us/new` for you. 

Would it make sense to carefully add this? Like, 
```
node tool/cli.js content move --no-git en-US web/accessibility/understanding_wcag Web/foobar2000
```
Perhaps overkill to worry about and it's cheap and simple to just set aside an hour to write the `child_process` thing that just does it.